components/director/pkg/inputvalidation/validators_cert_subject_mappings.go

```diff
@@ -25,6 +25,8 @@ const (
 	ManagedApplicationConsumerType = "Managed Application Consumer"
 	// LandscapeResourceOperatorType represents a landscape resource operator consumer type
 	LandscapeResourceOperatorType = "Landscape Resource Operator"
+	// TenantDiscoveryOperatorType represents a tenant discovery operator consumer type
```

Comment:
One more place needed to be adapted - `components/director/pkg/inputvalidation/validators_cert_subject_mappings.go` It's a consumer type validation when certificate subject mappings are created. I guess you will create CSM for this operator?

P.S. I'm not sure why all the consumers are duplicated in the mentioned file, I'll check that and make a PR to fix it and use the ones from the consumer package.